agents/architect/Dockerfile

```
@@ -0,0 +1,39 @@
+COPY package.json package-lock.json ./
+RUN npm ci
+
+# Copy source and compile TypeScript → dist/
+COPY tsconfig*.json ./
+COPY src/ src/
+RUN npm run build
+
+# ── Stage 2: Runtime ───────────────────────────────────────────────────────
+FROM node:20-alpine AS runtime
+
+RUN apk add --no-cache curl
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)